contracts/VaultV3.vy

```diff
@@ -654,3 +665,10 @@ def accept_role_manager():
     assert msg.sender == self.future_role_manager
     self.role_manager = msg.sender
     self.future_role_manager = ZERO_ADDRESS
+
+@external
+def shutdown_vault():
```

Comment:
The current implementation only allows to shutdown the vault, it cannot be reactivated. 

Was this on purpose? Should we allow reactivation? Any reason not to?

Reply:
also, I think it's important that we keep using camel case. It's weird to use camelcase in a pythonic lang but I think it's even weirdeir to use snake_case in smart contracts.

Reply:
We agreed on shutdown is definitive so attack surface is smaller. 

This may be something to reconsider in the future.